src/main/scala/mesosphere/marathon/tasks/TaskUrlUtil.scala

```diff
@@ -0,0 +1,20 @@
+package mesosphere.marathon.tasks
+
+import com.google.inject.Inject
+import mesosphere.marathon.MarathonConf
+import mesosphere.marathon.Protos.MarathonTask
+import mesosphere.util.state.{ FrameworkIdUtil, MesosMasterUtil }
+
+class TaskUrlUtil @Inject() (
```

Comment:
I like that you split this out into a separate class!

Please add a short description.